resources/icons/16px/subscript.svg

```diff
@@ -0,0 +1,3 @@
+<svg width="17" height="16" viewBox="0 0 17 16" fill="none" xmlns="http://www.w3.org/2000/svg">
```

Comment:
@davidburns573 there's an issue with both 16px icons because of this 17px viewbox. they get clipped horizontally:

<img width="85" alt="image" src="https://user-images.githubusercontent.com/723999/232559480-6b24fcdb-2eb8-49e7-a5a5-1a6864370851.png">

can you please fix this in a follow-up PR? I will likely hold release of this package until this is fixed.